railties/test/application/per_request_digest_cache_test.rb

```diff
@@ -34,6 +34,7 @@ def cache_key
     app_file "app/controllers/customers_controller.rb", <<-RUBY
       class CustomersController < ApplicationController
         self.perform_caching = true
+        Rails.application.config.action_view.render_hints = true
```

Comment:
we should set value back to the original one and we should not set a config in a controller.

Reply:
Corrected in the added tests in `actionview/test/template/render_test.rb`.